src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+public static class MicrobeInternalCalculations
+{
+    public static float CalculateSpeed(IEnumerable<OrganelleTemplate> organelles, MembraneType membraneType,
+        float membraneRigidity)
+    {
+        float microbeMass = Constants.MICROBE_BASE_MASS;
+
+        float organelleMovementForce = 0;
+
+        foreach (var organelle in organelles)
+        {
+            microbeMass += organelle.Definition.Mass;
+
+            if (organelle.Definition.HasComponentFactory<MovementComponentFactory>())
+            {
+                if (organelle.Orientation == 3)
```

Comment:
This is likely due to the implementation of organelles, but checking if an orientation equals 3 utterly confuses me.

Reply:
I added a comment to try to help. It's a sloppy way to ensure that neither the editor nor auto-evo factors in forward-facing flagella for speed calculations.

Reply:
Well this is actually entirely incorrect. The flagella thrust is entirely dependent on where it ends up being with the external organelle positioning. The orientation has 0 effect on the actual force it can generate...